packages/tc-api-rest-handlers/lib/events/__tests__/events-integration.spec.js

```diff
@@ -226,6 +225,31 @@ describe('Rest events module integration', () => {
 			]);
 			expectUpdateEvent(childType, childCode, ['isChildOf']);
 		});
+
+		it('should not send extra UPDATE events when removing rich relationship property that was already non-existent', async () => {
+			const [main, child] = await Promise.all([
+				createMainNode(),
+				createNode(childType, { code: childCode }),
+			]);
+			await connectNodes(main, 'HAS_CURIOUS_CHILD', child);
+
+			const { status } = await patchHandler()(
+				getInput(
+					{
```

Comment:
getInput is a helper that return an object of the form `{code, type, body, metadata, query}`... i.e. the input for the rest handler functions